docs/porting.md

```diff
@@ -153,6 +153,21 @@ after importing a module that does not support the GIL.
         endif
         ```
 
+=== "f2py"
+
+    Starting with NumPy 2.1.0 (only available via the nightly wheels or the
+    `main` branch as of right now), Fortran extension modules wrapped using
+    f2py can declare they are thread safe and support free-threading using the
+    [`--freethreading-compatible`](https://numpy.org/devdocs/f2py/usage.html#extension-module-construction)
+    command-line argument:
+
+    ```bash
+    $ python -m numpy.f2py -c code.f -m my_module --freethreading-compatible
+    ```
+
+    If you know the extension is *not* thread safe, then you can force the
```

Comment:
This should not be necessary right, since that's the default already? If so, then it's better to leave this out I think.

Or do you recommend this for clarity only? If so, I suggest saying that explicitly.